### DISCUSSION TITLE
Easy through-hole version of OSHW design...

### DISCUSSION THREAD

**Discussion by radix50** [ZuluSCSI/ZuluSCSI-Pico-OSHW#5] [Show and tell]
Hi All
Firstly, thanks for a great product in ZuluSCSI and everything it is based on...  

I was please to find the OSHW version - but was looking for a simple design with no surface mount parts - so I have been testing a prototype and am currently waiting for some boards (render below, hopefully!)

I would like to GIVE this project to the community, for the benefit of all, so hobbyists like myself can build a basic SD-SCSI device at minimal cost, with easy to solder parts.

Could you please advise the best way to do this, I would prefer it to become part of your OSHW offering if that is an option please.

![image](https://github.com/user-attachments/assets/77d92972-8c9f-4dff-afbf-4d7d4a208762)

**Comment by radix50** [comment]
Oh yes, I'm Robin, from the UK - currently volunteering at the National Museum of Computing, working on vintage DEC and other systems, where the ZuluSCSI cards are REALLY useful!

**Comment by aperezbios** [comment]
@radix50 Cool! could you share the BOM for this? I'm mostly interested in which DIP-form-factor parts you selected for the transceivers. As far as incorporating the design in to this repository, perhaps it would be simplest if it were to live in a branch? I would prefer not to merge it until you know it works :) When do you expect to receive the PCBs?

**Reply by radix50** [reply]
Hi, Yes, the choice of transceiver chips was the challenge! - so - you can use 74LS641-1 chips for the 8-bit buffers, these are bi-directional open collector drivers, so they can level shift either way to 3v3 with pull-ups, and 7406/7407 open collector buffers and inverters to replace the 74LVT125 parts - this does mean you cannot do initiator mode in the same way, so it's a simple target only board.

The veroboard (prototyping board) prototype works perfectly on an Adaptec 19160 on a PC and on two DEC SCSI controllers in a PDP-11, so looking good at present...  Boards will be a week or two I expect, them more testing.  

I think I should hold off publishing the complete design until I get the official OK, hence the post here in the first place.

Robin

**Reply by ZigZagJoe** [reply]
How do you handle the termination?

**Reply by aperezbios** [reply]
@ZigZagJoe the PCB silkscreen seems to indicate that there isn't any termination, but I'm not 100% sure. @radix50 would you be willing to upload a slightly higher-resolution render of the board?

**Reply by radix50** [reply]
Hi - Termination uses a proper terminator resistor network in a socket, 20 pin, 18 pairs of resistors, so exactly enough for all lines - remove chip to disable termination..

These were used on many real SCSI devices in the past...

**Reply by radix50** [reply]
Form factor is 3.5" drive size with same mounting holes, 
Ignore the "Viking SCSI" connector - there's a SCSI controller that uses a couple of pins for a serial interface to configure it (actual RS-232 levels as well!)

If there are any qualification tests you run to test the boards, I would be interested...

Robin

![image](https://github.com/user-attachments/assets/4746d11b-8292-4f84-b65a-cbbc1c17d952)

**Reply by aperezbios** [reply]
@radix50 I'm looking forward to trying one of these out, and it's great that you've got it working on veroboard! Can you e-mail me at support@zuluscsi.com?

**Reply by radix50** [reply]
Dropped you an email - Thanks!

**Reply by aperezbios** [reply]
> If there are any qualification tests you run to test the boards, I would be interested...

I would start by using a ZuluSCSI in initiator mode. Do you a production ZuluSCSI board at present? If not, let me know.

**Reply by radix50** [reply]
Hi - Yes, I have bought 2 of the full size ZuluSCSI boards - but I need more for different projects...

**Reply by aperezbios** [reply]
> Dropped you an email - Thanks!

I sent you a response. Did you receive it?

**Comment by radix50** [comment]
Oh there's a cheap micro-SD card adapter board not shown in the render to deal with the card socket...
